web_src/js/index.js

```diff
@@ -901,25 +901,23 @@ async function initRepository() {
       const target = $(this).data('target');
       const quote = $(`#comment-${target}`).text().replace(/\n/g, '\n> ');
       const content = `> ${quote}\n\n`;
-
-      let $content;
+      let $simplemde = autoSimpleMDE;
       if ($(this).hasClass('quote-reply-diff')) {
         const $parent = $(this).closest('.comment-code-cloud');
         $parent.find('button.comment-form-reply').trigger('click');
-        $content = $parent.find('[name="content"]');
-        if ($content.val() !== '') {
-          $content.val(`${$content.val()}\n\n${content}`);
-        } else {
-          $content.val(`${content}`);
-        }
-        $content.focus();
-      } else if (autoSimpleMDE !== null) {
-        if (autoSimpleMDE.value() !== '') {
-          autoSimpleMDE.value(`${autoSimpleMDE.value()}\n\n${content}`);
+        $simplemde = $parent.find('[name="content"]').data('simplemde');
+      }
+      if ($simplemde !== null) {
+        if ($simplemde.value() !== '') {
+          $simplemde.value(`${$simplemde.value()}\n\n${content}`);
         } else {
-          autoSimpleMDE.value(`${content}`);
+          $simplemde.value(`${content}`);
         }
       }
+      setTimeout(() => {
```

Comment:
Note that `setTimeout` can be pretty slow. I recommend using `requestAnimationFrame` where possible, or better yet, avoid these both altogether.

Reply:
Ok, I changed it to `requestAnimationFrame`. Without the setTimeout/requestAnimationFrame, the focus goes to the body element (checked with `document.activeElement`). I think it has something to do with the dropdown menu closing, but can't really afford to spend more time investigating that.

Reply:
Focus going to body usually means something is calling `.blur` and I think it's probably coming from fomantic so we probably can't avoid it.

https://github.com/fomantic/Fomantic-UI/blob/develop/src/definitions/modules/dropdown.js